From e41b7c10c4a998cbd812a539f2fe653384eaac56 Mon Sep 17 00:00:00 2001
From: Bao Xiahong <b50906@freescale.com>
Date: Tue, 14 Jul 2015 13:43:54 +0800
Subject: [PATCH 513/602] MA-6867 add ape parser

update imx6.mk & imx7.mk, add lib_ape_parser_arm11_elinux.3.0

Signed-off-by: Bao Xiahong <b50906@freescale.com>
---
 imx6/imx6.mk | 1 +
 imx7/imx7.mk | 1 +
 2 files changed, 2 insertions(+)

diff --git a/imx6/imx6.mk b/imx6/imx6.mk
index cb08597..ca4d06c 100644
--- a/imx6/imx6.mk
+++ b/imx6/imx6.mk
@@ -259,6 +259,7 @@ omx_libs := \
     libavresample \
 	lib_omx_libav_audio_dec_arm11_elinux \
     lib_omx_soft_hevc_dec_arm11_elinux \
+    lib_ape_parser_arm11_elinux.3.0 \
 
 
 
diff --git a/imx7/imx7.mk b/imx7/imx7.mk
index c374ba4..53793c2 100644
--- a/imx7/imx7.mk
+++ b/imx7/imx7.mk
@@ -256,6 +256,7 @@ omx_libs := \
     libavresample \
 	lib_omx_libav_audio_dec_arm11_elinux \
     lib_omx_soft_hevc_dec_arm11_elinux \
+    lib_ape_parser_arm11_elinux.3.0 \
 
 
 
-- 
1.8.0

